notebooks/patient_readmission/patient-readmission-train.ipynb

```diff
@@ -1016,7 +1073,7 @@
    "source": [
     "X = df.drop('readmitted',axis=1)\n",
     "y = df['readmitted']\n",
-    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20, random_state=0)\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X.values, y.values, test_size=0.20, random_state=0)\n",
```

Comment:
A side effect of re-running w/ Python 3.8 is that `scikit-learn>=1.0` was used. In version 1.0+ there is a warning emitted by models when you mix pandas/numpy types between training & predictions. For example: 
  ```
   X does not have valid feature names, but StandardScaler was fitted with feature names
  ```
    The fix was to tweak the notebooks to use `numpy` types throughout and/or use Certifai's builtin the `CatEncoder` (in `certifai.common.utils.encoding`), instead of the duplicate copies in this repo (e.g. `notebooks/utils/cat_encoder.py`).  This applied to multiple notebooks, but didn't have a significant change.